tests/system/providers/google/cloud/bigtable/example_bigtable.py

```diff
@@ -63,24 +63,35 @@
 from airflow.utils.trigger_rule import TriggerRule
 from tests.system.providers.google import DEFAULT_GCP_SYSTEM_TEST_PROJECT_ID
 
-ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID").lower().replace('_', '-')
+
 PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT") or DEFAULT_GCP_SYSTEM_TEST_PROJECT_ID
-DAG_ID = "bigtable"
+DAG_ID = f"bigtable"
 
+CB_INSTANCE_ID_ALLOWED_MAX_LENGTH = 33  # Length should be between [6,33]
+CB_CLUSTER_ID_ALLOWED_MAX_LENGTH = 30  # Cluster-id Length should be between [6,30]
+CB_TABLE_ID_ALLOWED_MAX_LENGTH = 50  # Table-id Length should be between [1,50]
 
-CBT_INSTANCE_ID = f"bigtable-instance-id-{ENV_ID}"
+CB_INSTANCE_ID_BASE = f"inst-id-"
+CB_INSTANCE_ID_BASE = (
+    f"{CB_INSTANCE_ID_BASE}{ENV_ID[:CB_INSTANCE_ID_ALLOWED_MAX_LENGTH-(len(CB_INSTANCE_ID_BASE) + 2)]}"
+)
```

Comment:
```suggestion
).replace("_", "-")
```
or vise versa, not sure which format should be here